src/test/java/com/mikhail/tarasevich/charcounter/provider/CharCounterImplTest.java

```diff
@@ -0,0 +1,47 @@
+package com.mikhail.tarasevich.charcounter.provider;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.util.LinkedHashMap;
+import org.junit.jupiter.api.Test;
+
+class CharCounterImplTest {
+
+	CharCounterImpl charCounterImpl = new CharCounterImpl();
```

Comment:
private and final all fields, where field must not change in runtime, if field can be changes in runtime just private